model_bakery/baker.py

```diff
@@ -660,6 +676,11 @@ def generate_value(self, field: Field, commit: bool = True) -> Any:
         `attr_mapping` and `type_mapping` can be defined easily overwriting the
         model.
         """
+        field_name = field.attname
+        if self._use_faker_generator and field_name in faker_generator_mapping:
```

Comment:
See other comment about `ImportError`. I think that if we fail to import `faker` we should output a warning message (when `_use_faker_generator` is `True`) and continue past this logic so that it does not break, but still gives the user feedback that they are trying to use an optional feature for which they do not have all the requirements.

Reply:
I think it's cleaner to fail when someone tries to use the feature without installing faker.
But not using the feature should still be possible without faker being installed.

moving the `from .faker_gen import faker_generator_mapping` inside this if clause should do the trick...?